### PR TITLE
Add LED mode to show low battery

### PR DESCRIPTION
**Pull request by ParkerMeyers**

### CODE REVIEW

File: src/main/java/bhs/devilbotz/Robot.java [hunk 1]
```diff
@@ -123,7 +124,13 @@ public void robotPeriodic() {
   @Override
   public void disabledInit() {
     robotContainer.resetRobotPosition();
-    robotContainer.setLEDMode(LEDModes.SET_LOADING);
+    // Wait for the voltage to appear
+    if (RobotController.getBatteryVoltage() < 12.5) {
```
Comment (BrownGenius) [inline src/main/java/bhs/devilbotz/Robot.java]: We should probably make this a constant in Constants.java

**Review by BrownGenius — DISMISSED** [review]
Looks good.  Just maybe put the low voltage value in a separate Constants.java file

**Review by BrownGenius — DISMISSED** [review]
Looks good.  The only thing that might be useful is to print the current voltage in the Alert.

File: src/main/java/bhs/devilbotz/Robot.java [hunk 2]
```diff
@@ -123,7 +124,13 @@ public void robotPeriodic() {
   @Override
   public void disabledInit() {
     robotContainer.resetRobotPosition();
-    robotContainer.setLEDMode(LEDModes.SET_LOADING);
+    // Wait for the voltage to appear
+    if (RobotController.getBatteryVoltage() < Constants.MIN_BATTERY_VOLTAGE) {
+      new Alert("Battery voltage is low", Alert.AlertType.WARNING).set(true);
```
Comment (ParkerMeyers) [inline src/main/java/bhs/devilbotz/Robot.java]:
```suggestion
      new Alert("Battery voltage is low: " + RobotController.getBatteryVoltage() + "v", Alert.AlertType.WARNING).set(true);
```

**Review by BrownGenius — APPROVED** [review]

**Pull request merged**